### PR TITLE
FEAT: Support OpenBuddy

### PR DESCRIPTION
**Pull request by codingl2k1**

### CODE REVIEW

File: xinference/model/llm/utils.py [hunk 1]
```diff
@@ -174,6 +174,9 @@ def get_prompt(
                 else:
                     ret += role + ": Let's think step by step."
             return ret
+        elif prompt_style.style_name == "INSTRUCTION":
```
Comment (UranusSeven) [inline xinference/model/llm/utils.py:177]: Multi-round conversation is not supported？

**Comment by UranusSeven** [comment]
Please add doc for this model.

**Comment by UranusSeven** [comment]
Also, README needs to be updated.

File: xinference/model/llm/utils.py (hunk 1 above)
Comment (codingl2k1) [reply xinference/model/llm/utils.py:177]: Yes. Multi-round is not good for translate.

**Comment by codingl2k1** [comment]
> Also, README needs to be updated.

Fixed. Thanks.

**Review by UranusSeven — APPROVED** [review]

**Pull request merged**